apps/wallet/src/modals/FirstTimePopup/index.tsx

```diff
@@ -0,0 +1,116 @@
+import { StyleSheet, TouchableOpacity, View } from 'react-native';
+import { modalActions, Text } from '@walless/gui';
+import { BlueCircleBackground } from '@walless/icons';
```

Comment:
I don't think this is an icon, right?

Btw, you can create a `CircleBackground` and passing the `Blue` color as a prop